frontend/src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -143,7 +177,10 @@ export const GroupCard = ({
                                         arrow
                                         describeChild
                                     >
-                                        <Badge>Not used</Badge>
+                                        <ConditionallyRender
```

Comment:
I'm not sure of the sanity of nesting `ConditionallyRender` but it does work

Reply:
Yep, should be fine 🙂